core/iwasm/common/wasm_native.c

```diff
@@ -414,6 +417,17 @@ wasm_native_init()
         return false;
 #endif /* WASM_ENABLE_LIBC_EMCC */
 
+#if WASM_ENABLE_WASI_NN != 0
+    n_native_symbols = get_wasi_nn_export_apis(&native_symbols);
+<<<<<<< HEAD
+    if (!wasm_native_register_natives("wasi_nn", native_symbols,
+=======
+    if (!wasm_native_register_natives("env", native_symbols, // TODO: check env or wasi_nn tag
+>>>>>>> 535a970550fc1780975ae9b320aa62c28318190d
+                                      n_native_symbols))
```

Comment:
Please check how you solved the conflicts!

Reply:
In addition to that. What are the 2 new commits adding to the code merge from the other branch?